db/migrate/20230519170746_create_search_attempts.rb

```diff
@@ -0,0 +1,15 @@
+class CreateSearchAttempts < ActiveRecord::Migration[6.0]
```

Comment:
The new keys are `bigint`s, and the old ones are just `int`s

Reply:
I'm surprised that the references worked, then.  If you change the migration header to [5.0] instead of [6.0], for the table creation/column creation migration files, that should fix things.

Reply:
Thanks, updated that.